modules/vba_documents/app/swagger/vba_documents/document_upload/v1/upload_attributes_swagger.rb

```diff
@@ -25,11 +25,21 @@ class UploadAttributesSwagger
             property :code do
               key :description, File.read(VBADocuments::Engine.root.join('app', 'swagger', 'vba_documents', 'document_upload', 'status_code_description.md'))
               key :type, :string
+              key :example, nil
             end
 
             property :detail do
               key :description, 'Human readable error detail. Only present if status = "error"'
               key :type, :string
+              key :example, ''
```

Comment:
These are the typical values for `code` and `detail` for the endpoint (`POST /uploads`) that uses this schema.